config/rbac/role.yaml

```diff
@@ -2,7 +2,6 @@
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  creationTimestamp: null
```

Comment:
Update here as well: config/hub/rbac/role.yaml (and config/dr-cluster/rbac/role.yaml)